app/code/Magento/Customer/Controller/Account/CreatePassword.php

```diff
@@ -42,25 +41,34 @@ class CreatePassword extends \Magento\Customer\Controller\AbstractAccount implem
      */
     private $confirmByToken;
 
+    /**
+     * @var \Magento\Customer\Model\ForgotPasswordToken\GetCustomerByToken
+     */
+    private $getByToken;
+
     /**
      * @param \Magento\Framework\App\Action\Context $context
      * @param \Magento\Customer\Model\Session $customerSession
      * @param \Magento\Framework\View\Result\PageFactory $resultPageFactory
      * @param \Magento\Customer\Api\AccountManagementInterface $accountManagement
      * @param \Magento\Customer\Model\ForgotPasswordToken\ConfirmCustomerByToken $confirmByToken
+     * @param \Magento\Customer\Model\ForgotPasswordToken\GetCustomerByToken $getByToken
```

Comment:
Seems like all listed arguments are already imported, and some of them are not always an object, so we can update it to this
```suggestion
     * @param Context $context
     * @param Session $customerSession
     * @param PageFactory $resultPageFactory
     * @param AccountManagementInterface $accountManagement
     * @param ConfirmCustomerByToken|null $confirmByToken
     * @param GetCustomerByToken|null $getByToken
```